examples/tfengine/full.yaml

```diff
@@ -63,9 +63,11 @@ templates:
       project_id: "example-monitor"
       domain: "example.com"
     org_policies:
-      parent_type: folder
+      parent_type: "organization" # one of `organization`, `folder` or `project`
       parent_id: "12345678"
     cicd:
+      parent_type: "organization" # one of `organization` or `folder`
```

Comment:
I believe you can also just set parent_type and parent_id as a sibling of org_policies/cicd. I think this will be cleaner as typically you will want the same parent for both. We can document in the org schema policies that it can also optionally set parent to project if needed.

Reply:
Done